doc/source/spelling_wordlist.txt

```diff
@@ -0,0 +1,2557 @@
+ipython
+numpy
+NumPy
+Reindexing
+reindexing
+ga
+fe
+reindexed
+Histogramming
+histogramming
+concat
+resampling
+iterables
+sparsified
+df
+loc
+Timeseries
+ndarrays
+ndarray
+dtype
+dtypes
+dtyped
+reindex
+sliceable
+timedelta
+Timedeltas
+timedeltas
+subpackages
+subpackage
+filepath
+io
+nthreads
+kwargs
+kwarg
+arg
+args
+Datetimelike
+datetime
+datetimes
+tz
+builtin
+NaN
+nan
+behaviour
+quantiling
+aggregators
+aggregator
+Dtypes
+groupby
+GroupBy
+Tablewise
+Elementwise
+ufunc
+ufuncs
+dict
+namedtuples
+namedtuple
+iterrows
+upcasted
+upcasting
+upcast
+searchsorted
+downcasting
+Likert
+categoricals
+Groupby
+Unioning
+csv
+Upcase
+resampling
+Upcase
+Lowcase
+Propcase
+Interop
+Stata
+stata
+bysort
+Spearman
+Wikipedia
+debiasing
+docstrings
+docstring
+Docstrings
+autosummary
+linting
+toolchain
+Appveyor
+Akogun
+online
+pdf
+reStructuredText
+reST
+backticks
+cpus
+str
+idxmin
+mins
+agg
+DataFrame
+dataframes
+NaT
+len
+Statsmodels
+Bokeh
+Protovis
+Seaborn
+Wickham
+shareability
+apps
+app
+Plotly
+Spyder
+Fama
+Eurostat
+organisations
+Geopandas
+Dask
+Scikit
+backends
+Engarde
+Cyberpandas
+Accessor
+Numba
+optimising
+Cython
+cython
+cythonizing
+cythonized
+Vectorize
+ol
+subclassing
+IPv
+iteritems
+itertuples
+dt
+upcast
+subsetting
+programmatically
+stderr
+scipy
+SparseArray
+doctests
+nd
+refactored
+Jit
+stdout
+Typeclass
+Pythonic
+zscore
+SQL
+broadcastable
+resample
+resamples
+groupbys
+metaprogramming
+upcast
+un
+dropna
+ints
+int
+boxplot
+groupwise
+indices
+pre
+datetimelike
+dev
+gd
+colname
+intemname
+nd
+isin
+backporting
+admin
+Debian
+Ubuntu
+Centos
+RHEL
+xlsx
+xz
+ftp
+impl
+timespans
+pre
+Regex
+regex
+sortedness
+delim
+usecols
+skipinitialspace
+skiprows
+skipfooter
+nrows
+na
+iso
+dayfirst
+chunksize
+gz
+bz
+lineterminator
+quotechar
+doublequote
+escapechar
+tupleize
+prepended
+colspecs
+NONNUMERIC
+serializer
+localhost
+json
+strtod
+deserialization
+Hadoop
+ns
+stringified
+xclip
+xsel
+gtk
+gtpy
+Msgpacks
+msgpack
+msgpacks
+foo
+ptrepack
+sqlalchemy
+sqlite
+Sqlite
+dta
+bdat
+netCDF
+backend
+deserialising
+deserializing
+qtpy
+indexables
+itemsize
+de
+sas
+Miniconda
+itemname
+ndims
+ndim
+mergands
+Timeseries
+timeseries
+asof
+Nans
+DataFrames
+fillna
+ffill
+bfill
+alignable
+sim
+py
+ipy
+colheader
+yearfirst
+repr
+EngFormatter
+frontends
+frontend
+longtable
+multirow
+cline
+clines
+colwidth
+Sparsify
+html
+pprint
+mathjax
+Jupyter
+xls
+xlsm
+hdf
+numexpr
+matplotlib
+Critchley
+Villanova
+del
+Hohmann
+Rychyk
+Buchkovsky
+Lenail
+Schade
+datetimeindex
+Aly
+Sivji
+Költringer
+Bui
+András
+Novoszáth
+Anh
+Anil
+Pallekonda
+Pitrou
+Linde
+Quinonez
+Varshokar
+Artem
+Bogachev
+Avi
+Azeez
+Oluwafemi
+Auffarth
+Thiel
+Bhavesh
+Poddar
+Haffner
+Naul
+Guinta
+Moreira
+García
+Márquez
+Cheuk
+Chitrank
+Dixit
+Catalfo
+Mazzullo
+Chwala
+Cihan
+Ceyhan
+Brunner
+Riemenschneider
+Dixey
+Garrido
+Sakuma
+Hirschfeld
+Adrián
+Cañones
+Castellano
+Arcos
+Hoese
+Stansby
+Kamau
+Niederhut
```

Comment:
Not saying we need to do this here but wonder if its cleaner to have a separate contributors file containing their names that could be pulled here in as well as into the docs